### DISCUSSION TITLE
gapi.client.people.SourceType is missing OTHER_CONTACT and DOMAIN_CONTACT

### DISCUSSION THREAD

**Discussion by oltreseba** [DefinitelyTyped/DefinitelyTyped#57742] [Issues with a @types package]
According to [google people API: ](https://developers.google.com/people/api/rest/v1/people#sourcetype) the list of SourceType is:
```
SOURCE_TYPE_UNSPECIFIED
ACCOUNT
PROFILE
DOMAIN_PROFILE
CONTACT
OTHER_CONTACT
DOMAIN_CONTACT
```
The last two looks missing in https://github.com/DefinitelyTyped/DefinitelyTyped/blob/35c79e6a031edc34628c4b8053cde6bd85311e7a/types/gapi.people/index.d.ts#L95

[a comment by an automated account is not shown]

**Reply by oltreseba** [reply]
Missing all SourceTypes in @types/gapi.people

**Comment by martin-badin** [comment]
PR is ready #57835
